recipe/meta.yaml

```diff
@@ -280,19 +294,31 @@ outputs:
         -     python setup.py install -v -v
         -     python -c "import foobar"
         -   popd
-        -   pushd prefix-replacement  # [unix]
-        -     bash build-and-test.sh  # [unix]
+        -   pushd embedding-interpreter  # [unix]
+        -     bash build-and-test.sh  # [unix and not (linux and aarch64)]
         -   popd  # [unix]
-        -   pushd cmake
-        -     cmake -GNinja -DPY_VER={{ version }}
-              # --trace --debug-output --debug-trycompile .
+        -   pushd cmake  # [not (linux and ppc64le)]
+        -     run_cmake_test.bat {{ version }}  # [win]
+        -     bash run_cmake_test.sh {{ version }}  # [not win and not (linux and ppc64le)]
+        -   popd  # [not (linux and ppc64le)]
+        -   pushd processpoolexecutor-max_workers-61
+        -     python ppe.py
         -   popd
         - popd
         - test ! -f default.profraw   # [osx]
 
+  - name: python-regr-testsuite
+    requirements:
+      host:
+        - {{ pin_subpackage('python', exact=True) }}
+    script: install_regr_testsuite.sh  # [unix]
+
+{% if os.getenv("build_platform") != "linux-aarch64"  %}
```

Comment:
Why?

Reply:
Are you asking why we want to make a split package for the python regression testsuite here? Before we had split packages, Jonathan felt that the test-suite added too much to the core interpreter, so this is a step in the direction towards being able to better test our releases.